src/test/java/api/support/builders/LostItemFeePolicyBuilder.java

```diff
@@ -204,6 +204,10 @@ public LostItemFeePolicyBuilder withChargeAmountItem(String chargeType, Double a
       .put("chargeType", chargeType));
   }
 
+  public LostItemFeePolicyBuilder withSetCost(Double amount) {
+    return withChargeAmountItem("anotherCost", amount);
+  }
```

Comment:
Have all usages of `withChargeAmountItem` been replaced with `withSetCost`, except for specific failure scenarios where the `chargeType` is not recognised?

Reply:
Yes, they are.